static/js/page/events/map/Map.js

```diff
@@ -67,19 +67,24 @@ export default class Map {
     });
 
     emitter.on(EVENTS.EVENT_HIGHLIGHTED, (event) => {
+      if (!event.hasCity()) return;
```

Comment:
may be we just don't be [emmit event](https://github.com/JetBrains/kotlin-web-site/blob/4604c9bb6623685b20121bbddae74a0d06252b2b/static/js/page/events/events-list/EventsList.js#L73) for this online events?

Reply:
There are two listeners subscribed to the "EVENT_HIGHLIGHTED" event: a list and a map. In the list, the event is used for the related item to highlight it. But not on the map. So I had to make the condition in the map listener.